ambry-router/src/main/java/com.github.ambry.router/RouterUtils.java

```diff
@@ -42,7 +42,7 @@ static BlobId getBlobIdFromString(String blobIdString, ClusterMap clusterMap)
       blobId = new BlobId(blobIdString, clusterMap);
       logger.trace("BlobId created " + blobId + " with partition " + blobId.getPartition());
     } catch (Exception e) {
-      logger.error("Caller passed in invalid BlobId " + blobIdString);
+      logger.trace("Caller passed in invalid BlobId " + blobIdString);
```

Comment:
@xiahome we have metrics for such errors. such user errors can populate the logs and mask real errors, so these are best left in trace.

@vgkholla trace vs. error does not really matter as this is only executed in the error scenario - but the format string is a good practice to follow anyway.